javaslang/src/main/java/javaslang/collection/PriorityQueue.java

```diff
@@ -544,210 +544,214 @@ public boolean equals(Object o) {
         return o == this || o instanceof PriorityQueue && Collections.areEqual(this, (Iterable) o);
     }
 
-    protected static class PriorityQueueBase {
-        /* Based on http://www.brics.dk/RS/96/37/BRICS-RS-96-37.pdf */
-        protected static class Node<T> implements Serializable {
-            private static final long serialVersionUID = 1L;
+}
```

Comment:
Sputnik complained about the order: method decls vs class decls. I re-ordered them according to the coding conventions.